### PR TITLE
Restrict charge tables to active bookings only

### LINKED ISSUE #21: User can see tabular summary for charge category
_From @hartsick on November 30, 2016 21:59_

Data sources
    * Charge category - charges table
    * People booked in last week - bookings table

Include counts, percentage of active bookings

_Copied from original issue: codeforamerica/jail-dashboard-project#136_

### LINKED ISSUE #23: User sees tabular summary for charge type
_From @hartsick on November 30, 2016 21:56_

Data sources:
    * Charge type: charges table
    * People booked in last week - bookings table
    * Charge description = charge type

Tabular data for charge type counts and percentage of active bookings

Use current Louisville site for reference. Current Rails app does not bound bookings by 'active'.

_Copied from original issue: codeforamerica/jail-dashboard-project#134_

### PR DESCRIPTION
**Pull request by graysonwright**
Closes #21.
Closes #23.

Problem
-------

The charge category and charge type tables
display counts for all charges in the database.

The rest of the page limits results to only active bookings,
so these tables display extra information that is out of context.

Solution
--------

Use [`ActiveRecord#merge`] to limit displayed charges to only those
belonging to active bookings.

[`ActiveRecord#merge`]: http://apidock.com/rails/ActiveRecord/SpawnMethods/merge

### CODE REVIEW

File: app/views/pages/home.html.haml [hunk 1]
```diff
@@ -32,14 +32,14 @@
       %h3 Charges Summary
       %br
       %h4 By Category
-      %table.table.table-striped.table-responsive
-        -@charges.group_by(&:category).each do |category, charges|
+      %table.table.table-striped.table-responsive.charge-categories
+        -@active_charges.group_by(&:category).each do |category, active_charges|
```
Comment (jennifermarie) [inline app/views/pages/home.html.haml]: What's our take on this? Worth enforcing or ignore?

Comment (graysonwright) [inline app/views/pages/home.html.haml]: I think it's worth enforcing - it seems more readable to me

Comment (jennifermarie) [inline app/views/pages/home.html.haml]: 👍

File: spec/factories/bookings.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 FactoryGirl.define do
   factory :booking do
-    jms_booking_id "98765"
+    sequence(:jms_booking_id) { |n| "%5d" % n }
```
Comment (jennifermarie) [inline spec/factories/bookings.rb]: http://batsov.com/articles/2013/06/27/the-elements-of-style-in-ruby-number-2-favor-sprintf-format-over-string-number-percent/

Comment (jennifermarie) [inline spec/factories/bookings.rb]: ^^ context for why Rubocop is probably picky about that

**Pull request merged**